gamestonk_terminal/portfolio/README.md

```diff
@@ -0,0 +1,40 @@
+#PORTFOLIO
+
+This was developed to interface with Robinhood.  
+
+Once this screen is accessed, the first command to be run is 
+````
+login
+````
+Which will use your robinhood login information, which should be stored as environment variables or in the config_terminal.py
```

Comment:
In this case you can add the config_terminal.py hyperlink, you can do so using:
```
[add here the name you want to show externally](hyperlink to whatever you want to show)
```

Reply:
I added GitHub  link to config_terminal.py.  Did you mean that or the local path?

Reply:
Yea, I mean doing this:

[config_terminal.py](/gamestonk_terminal/config_terminal.py)